backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/FeatureSupported.java

```diff
@@ -411,4 +412,16 @@ public static boolean isParallelMigrationsSupported(Version version) {
     public static boolean isConvertDiskSupported(VDS vds) {
         return vds != null && Version.v4_6.less(vds.getClusterCompatibilityVersion());
     }
+
+    /**
+     * Checks if the hybrid backup can be performed on VM
+     *
+     * @param vm The VM
+     * @return true if VM can be backed up using hybrid backup
+     */
+    public static boolean isHybridBackupSupported(VM vm) {
+        return vm != null &&
+                Version.v4_7.lessOrEquals(vm.getCompatibilityVersion()) &&
+                supportedInConfig(ConfigValues.UseHybridBackup, vm.getCompatibilityVersion());
```

Comment:
good catch, that's something I wanted to comment on and forgot..
then we can drop line 424